File: HEUR-Trojan/Win32/S/HEUR-Trojan.Win32.SelfDel.gen-525be95309216f0b535e9fffe96001a74436d6fa39dd91918d03a91211f831bc/Notepad/My/Resources/Resources.vb
Imports System
Imports System.CodeDom.Compiler
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Globalization
Imports System.Resources
Imports System.Runtime.CompilerServices
Imports Microsoft.VisualBasic
Imports Microsoft.VisualBasic.CompilerServices

Namespace vHack_Loader.My.Resources
	' Token: 0x02000005 RID: 5
	<GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")>
	<DebuggerNonUserCode()>
	<CompilerGenerated()>
	<HideModuleName()>
	Friend NotInheritable Module Resources
		' Token: 0x17000006 RID: 6
		' (get) Token: 0x0600000B RID: 11 RVA: 0x00002116 File Offset: 0x00000316
		<EditorBrowsable(EditorBrowsableState.Advanced)>
		Friend ReadOnly Property ResourceManager As ResourceManager
			Get
				If Object.ReferenceEquals(Resources.resourceMan, Nothing) Then
					Resources.resourceMan = New ResourceManager("vHack_Loader.Resources", GetType(Resources).Assembly)
				End If
				Return Resources.resourceMan
			End Get
		End Property

		' Token: 0x17000007 RID: 7
		' (get) Token: 0x0600000C RID: 12 RVA: 0x00002148 File Offset: 0x00000348
		' (set) Token: 0x0600000D RID: 13 RVA: 0x0000214F File Offset: 0x0000034F
		<EditorBrowsable(EditorBrowsableState.Advanced)>
		Friend Property Culture As CultureInfo
			Get
				Return Resources.resourceCulture
			End Get
			Set(value As CultureInfo)
				Resources.resourceCulture = value
			End Set
		End Property

		' Token: 0x17000008 RID: 8
		' (get) Token: 0x0600000E RID: 14 RVA: 0x00002157 File Offset: 0x00000357
		Friend ReadOnly Property _7 As Byte()
			Get
				Return CType(RuntimeHelpers.GetObjectValue(Resources.ResourceManager.GetObject("_7", Resources.resourceCulture)), Byte())
			End Get
		End Property

		' Token: 0x17000009 RID: 9
		' (get) Token: 0x0600000F RID: 15 RVA: 0x00002177 File Offset: 0x00000377
		Friend ReadOnly Property _8 As Byte()
			Get
				Return CType(RuntimeHelpers.GetObjectValue(Resources.ResourceManager.GetObject("_8", Resources.resourceCulture)), Byte())
			End Get
		End Property

		' Token: 0x1700000A RID: 10
		' (get) Token: 0x06000010 RID: 16 RVA: 0x00002197 File Offset: 0x00000397
		Friend ReadOnly Property Clean As Byte()
			Get
				Return CType(RuntimeHelpers.GetObjectValue(Resources.ResourceManager.GetObject("Clean", Resources.resourceCulture)), Byte())
			End Get
		End Property

		' Token: 0x1700000B RID: 11
		' (get) Token: 0x06000011 RID: 17 RVA: 0x000021B7 File Offset: 0x000003B7
		Friend ReadOnly Property cloud As Byte()
			Get
				Return CType(RuntimeHelpers.GetObjectValue(Resources.ResourceManager.GetObject("cloud", Resources.resourceCulture)), Byte())
			End Get
		End Property

		' Token: 0x1700000C RID: 12
		' (get) Token: 0x06000012 RID: 18 RVA: 0x000021D7 File Offset: 0x000003D7
		Friend ReadOnly Property MrsnapzNet As Byte()
			Get
				Return CType(RuntimeHelpers.GetObjectValue(Resources.ResourceManager.GetObject("MrsnapzNet", Resources.resourceCulture)), Byte())
			End Get
		End Property

		' Token: 0x04000006 RID: 6
		Private resourceMan As ResourceManager

		' Token: 0x04000007 RID: 7
		Private resourceCulture As CultureInfo
	End Module
End Namespace

File: HEUR-Trojan-Ransom/MSIL/E/HEUR-Trojan-Ransom.MSIL.Encoder.gen-c2bc6afd359591e309ee686069fbbef939d91defb130c21bd0ef59960fa4e79c/RANSOMWARE3._0.Properties/Resources.cs
using System.CodeDom.Compiler;
using System.ComponentModel;
using System.Diagnostics;
using System.Drawing;
using System.Globalization;
using System.Resources;
using System.Runtime.CompilerServices;

namespace RANSOMWARE3._0.Properties;

[GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]
[DebuggerNonUserCode]
[CompilerGenerated]
internal class Resources
{
	private static ResourceManager resourceMan;

	private static CultureInfo resourceCulture;

	[EditorBrowsable(EditorBrowsableState.Advanced)]
	internal static ResourceManager ResourceManager
	{
		get
		{
			if (resourceMan == null)
			{
				ResourceManager resourceManager = (resourceMan = new ResourceManager("RANSOMWARE3._0.Properties.Resources", typeof(Resources).Assembly));
			}
			return resourceMan;
		}
	}

	[EditorBrowsable(EditorBrowsableState.Advanced)]
	internal static CultureInfo Culture
	{
		get
		{
			return resourceCulture;
		}
		set
		{
			resourceCulture = value;
		}
	}

	internal static Bitmap Bez_tytułu
	{
		get
		{
			//IL_0017: Unknown result type (might be due to invalid IL or missing references)
			//IL_001d: Expected O, but got Unknown
			object @object = ResourceManager.GetObject("Bez tytułu", resourceCulture);
			return (Bitmap)@object;
		}
	}

	internal static byte[] BSOD
	{
		get
		{
			object @object = ResourceManager.GetObject("BSOD", resourceCulture);
			return (byte[])@object;
		}
	}

	internal static Bitmap i
	{
		get
		{
			//IL_0017: Unknown result type (might be due to invalid IL or missing references)
			//IL_001d: Expected O, but got Unknown
			object @object = ResourceManager.GetObject("i", resourceCulture);
			return (Bitmap)@object;
		}
	}

	internal static Bitmap i1
	{
		get
		{
			//IL_0017: Unknown result type (might be due to invalid IL or missing references)
			//IL_001d: Expected O, but got Unknown
			object @object = ResourceManager.GetObject("i1", resourceCulture);
			return (Bitmap)@object;
		}
	}

	internal Resources()
	{
	}
}